acquia_lift.admin.wizard.inc

```diff
@@ -344,6 +343,57 @@ function acquia_lift_personalize_campaign_wizard_variations_alter(&$form, &$form
       'wrapper' => $new_option_sets_id,
     ),
   );
+
+  // The rest of the display is for displaying current variations.
+  if (count($option_sets) == 0) {
+    return;
+  }
+
+  // Display all the current variation options.
+  $form['variations']['display'] = array(
+    '#type' => 'container',
+    '#attributes' => array(
+      'class' => array('personalize-wizard-column'),
+    ),
+  );
+
+  // If there are any errors in the multiple variation set handling for this
+  // agent then show them here while the user can do something about it.
+  if (count($option_sets) > 1) {
+    $agent_instance = personalize_agent_load_agent($agent_data->machine_name);
+    $errors = $agent_instance->getMultipleOptionSetErrors($option_sets);
```

Comment:
The class is already littered with external function calls so there's nothing to be gained by not adding one here - it's still not going to be unit-testable. This logic refers to a restriction that we are imposing at a higher level than just this plugin class. There's probably an argument to be made that none of this errors() stuff belongs in the class but should just be a hook, but that's for another day :P